### PR TITLE
Setup Cypress configuration

### PR DESCRIPTION
**Pull request by YesidCodes**

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (kliver98) [inline cypress/support/commands.js]: Add an enter at the end, EOF
Should have 26 lines and this last one without spaces

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (kliver98) [inline cypress/support/e2e.js]: Add an enter at the end, EOF
Should have 26 lines and this last one without spaces

Comment (YesidCodes) [reply cypress/support/e2e.js]: Done

File: cypress/support/commands.js (hunk 1 above)
Comment (YesidCodes) [reply cypress/support/commands.js]: Done

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**